src/operator/pairwise.ts

```diff
@@ -8,12 +8,13 @@ import {Subscriber} from '../Subscriber';
  * The (N-1)th is stored in the internal state until Nth input occurs.
  * @returns {Observable<R>} an observable of pairs of values.
  */
-export function pairwise<T>(): Observable<T> {
-  return this.lift(new PairwiseOperator());
+export function pairwise<T>(): Observable<[T, T]> {
+  let _this: Observable<T> = this;
```

Comment:
@kwonoj That's correct. David raised the performance aspect of this as well. Just so we can make this explicit, the concern is that the instance of `Observable` would be disposed by the GC if not for the `_this`/`self` reference, so adding this alias will result in memory pressure/a full out leak. Is this a fair assessment?

I will try to set up a test in Chrome memory mode to see how the alias affects frequency of GC and memory usage.